### ISSUE TITLE
[WARN] Methode overwrite conflict 

### ISSUE THREAD

**Issue by MrLoLf** [Jerozgen/LanguageReload#30]
I am using the latest version 1.5.4 on Minecraft 1.19.3 it posts in the [latest.log](https://github.com/Jerozgen/LanguageReload/files/10517418/latest.log) 
```
Method overwrite conflict for getChunks in languagereload.mixins.json:ClientChunkManagerAccessor from mod languagereload, previously written by me.cominixo.betterf3.mixin.chunk.ClientChunkManagerMixin. Skipping method.

[10:13:29] [Render thread/WARN]: Method overwrite conflict for getProgress in languagereload.mixins.json:AdvancementWidgetAccessor from mod languagereload, previously written by de.guntram.mcmod.advancementinfo.mixin.AdvancementWidgetMixin. Skipping method.

[10:13:29] [Render thread/WARN]: Method overwrite conflict for getChunks in languagereload.mixins.json:ClientChunkMapAccessor from mod languagereload, previously written by me.cominixo.betterf3.mixin.chunk.ClientChunkMapMixin. Skipping method.
``` 

this doesn't occur when the mod is used alone, as it looks like it's overwriting stuff from other mods.

**Comment by Jerozgen** [comment]
This should be fixed in 1.5.5

**Comment by PlayfulPiano** [comment]
While this is a bit old, could it be possible to backport this fix for the 1.18.2 version of languagereload?

**Comment by Jerozgen** [comment]
Sorry, I won't backport that to 1.18.2
